CheatSheets/html-cheatsheet.md

```diff
@@ -145,3 +152,19 @@
 | `<col>`                    | Denotes a column inside a table.                                   |
 
 **[🔼Back to Top](#table-of-contents)**
+
+## Graphics
+
+| Command                    | Description                                                        |
+| `<canvas> … </canvas>`     | Used to draw graphics on a web page using javascript.              |
+| `<svg> … </svg>`           | Used to defines vector-based graphics in XML format.               |
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Media
+
+| Command                    | Description                                      |
+| `<video> … </video>`       | Used to show a video on a web page.              |
+| `<audio> … </audio>`       | Used to play an audio file on a web page.        |
+
+**[🔼Back to Top](#table-of-contents)**
```

Comment:
```suggestion
## Graphics

| Command                    | Description                                                        |
| --------------------- | ------------------- |
| `<canvas> … </canvas>`     | Used to draw graphics on a web page using javascript.    |
| `<svg> … </svg>`           | Used to defines vector-based graphics in XML format.               |

**[🔼Back to Top](#table-of-contents)**

## Media

| Command                    | Description                                      |
| --------------------- | ------------------- |
| `<video> … </video>`       | Used to show a video on a web page.              |
| `<audio> … </audio>`       | Used to play an audio file on a web page.        |

**[🔼Back to Top](#table-of-contents)**
```
The table is broken again. [here](https://github.com/dotsbeauty/CheatSheets-for-Developers/blob/main/CheatSheets/html-cheatsheet.md#graphics)